codalab/server/bundle_manager.py

```diff
@@ -707,8 +721,13 @@ def _get_staged_bundles_to_run(self, workers, user_info_cache):
             failures.append(
                 self._check_resource_failure(
                     bundle_resources.disk,
-                    user_fail_string='Requested more disk (%s) than user disk quota left (%s)',
-                    user_max=self._model.get_user_disk_quota_left(bundle.owner_id, user_info),
+                    user_fail_string='Requested more disk (%s) than user disk quota left (%s) by %s bytes',
```

Comment:
Can we pretty print the difference too instead of using bytes?

Reply:
Yes, we can. When using pretty print, the final error message won't contain the unit `bytes`, which could be a little bit confusing. See the logic here: [size_str](https://github.com/codalab/codalab-worksheets/blob/b8b2efed243150dfd8398d4a6136b3d351bcb9fa/codalab/lib/formatting.py#L42). We could add `b` to `size_str` function, or if we think keeping the formatting as it's is fine, I'll update the function to use pretty print.

Reply:
@percyliang Want to double-check with you on ^ options. Let me know!